test/Interop/Cxx/foreign-reference/invalid-retain-operation-errors.swift

```diff
@@ -28,6 +28,44 @@ BadRetainRelease {};
 int badRetain(BadRetainRelease *v);
 void badRelease(BadRetainRelease *v, int i);
 
+struct
+    __attribute__((swift_attr("import_reference")))
+    __attribute__((swift_attr("retain:goodRetain")))
+    __attribute__((swift_attr("release:goodRelease")))
+GoodRetainRelease {};
+
+void goodRetain(GoodRetainRelease *v);
+void goodRelease(GoodRetainRelease *v);
+
+struct
+    __attribute__((swift_attr("import_reference")))
+    __attribute__((swift_attr("retain:badRetain2")))
+    __attribute__((swift_attr("release:badRelease2")))
+BadRetainRelease2 {};
+
+void badRetain2(int);
+void badRelease2(int);
+
+struct
+    __attribute__((swift_attr("import_reference")))
+    __attribute__((swift_attr("retain:badRetain3")))
+    __attribute__((swift_attr("release:badRelease3")))
+BadRetainRelease3 {};
+
+void badRetain3(BadRetainRelease2 *);
+void badRelease3(BadRetainRelease2 *);
+
+struct nonCXXFRT{};
+
+struct
+    __attribute__((swift_attr("import_reference")))
+    __attribute__((swift_attr("retain:badRetain4")))
+    __attribute__((swift_attr("release:badRelease4")))
+BadRetainRelease4 {};
+
+void badRetain4(nonCXXFRT *);
+void badRelease4(nonCXXFRT *);
+
```

Comment:
Could you please add a test where retain/release operations have nullability annotations, e.g. `_Nullable`/`_Nonnull`